dev-docs/bidders/roundhouseads.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: bidder
+title: RoundhouseAds
+description: RoundhouseAds Prebid Bidder Adapter
+biddercode: roundhouseads
+tcfeu_supported: false
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+media_types: banner, video, native
+floors_supported: true
+pbjs: true
+sidebarType: 1
```

Comment:
This document is missing quite a few pieces of adapter metadata. Please fill out the metadata as documented at https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter